### PR TITLE
intial changes to cloud starter guide

### PR DESCRIPTION
**Pull request by ashleyjanedoiron**
Initial changes to the cloud starter guide found here (https://tina.io/guides/tina-cloud/starter/overview/)

### CODE REVIEW

File: content/guides/tina-cloud/starter/create-account.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+## TinaCMS Backend
+
+While the fully local development workflow is the recommended way for developers to work,
+you'll likely want other editors and collaborators to be able to make changes on a hosted website with authentication. Since we are aiming for collaboration, we will use the TinaCMS dashboard to setup our site and eventually invite other users.
+
+## Create an Account
+
+1. Visit <a href="https://auth.tina.io/register" target="_blank">auth.tina.io</a>, create an account, and signin.
```
Comment (DirtyF) [inline content/guides/tina-cloud/starter/create-account.md:13]:
```suggestion
1. Visit <a href="https://app.tina.io/register" target="_blank">auth.tina.io</a>, create an account, and signin.
```

Shouldn't it be the new URL here?

File: content/guides/tina-cloud/starter/structure.md [hunk 2]
```diff
@@ -111,4 +114,4 @@ Tina Cloud generates your GraphQL schema automatically. 🪄
 
 ### Explore the GraphQL API
 
-If you have a GraphQL client like [Altair](https://altair.sirmuel.design/) go to `http://localhost:4001/graphql` to browse the docs and query our GraphQL API.
\ No newline at end of file
+If you have a GraphQL client like [Altair](https://altair.sirmuel.design/) go to `http://localhost:4001/graphql` to browse the docs and query our GraphQL API.
```
Comment (DirtyF) [inline content/guides/tina-cloud/starter/structure.md:117]: We now embed Altair by default.

```suggestion
To browse the docs and query our GraphQL API, run 'yarn dev` in your Tina project and go to `http://localhost:4001/altair`.
```

**Review by jeffsee55 — DISMISSED** [review]

**Comment by perkinsjr** [comment]
@ashleyjanedoiron looks great, we can merge this tomorrow.

**Comment by jamespohalloran** [comment]
> @ashleyjanedoiron looks great, we can merge this tomorrow.

Could this be merged now? I'm fine waiting, but it might help us avoid stepping on toes as we test things out today.

**Comment by perkinsjr** [comment]
If the new starter is out we can absolutely do that.

**Comment by jamespohalloran** [comment]
Thoughts @spbyrne ?

**Comment by logan-anderson** [comment]
@jamespohalloran @perkinsjr The new stater is already out

**Comment by ashleyjanedoiron** [comment]
> > @ashleyjanedoiron looks great, we can merge this tomorrow.
> 
> Could this be merged now? I'm fine waiting, but it might help us avoid stepping on toes as we test things out today.

I believe @jeffsee55  is working to merge his branch (https://github.com/tinacms/tinacms.org/pull/951) into this one. And then I'm going to add in a few notes from @logan-anderson's comments from earlier today. And then this will be good to go.

**Comment by logan-anderson** [comment]
On the deploy page

> Connect to your GitHub repository and set the same environment variables as the ones in your env.local file:
> `NEXT_PUBLIC_TINA_CLIENT_ID= <YOUR_CLIENT_ID>`
> 🎉 Congratulations, your site is now live!

Only doing this doesn’t seem to work I was getting a lot of errors.
We also need to
1.  update the Tina Cloud app URL to match the production URL. (OR  create a new app)
2. Add another env var on vercel
```
NEXT_PUBLIC_USE_LOCAL_CLIENT=0
```
Once I did this I was able to edit my production site

**Comment by logan-anderson** [comment]
We also didn’t talk about media. Since this guide is going to take place of the readme should myself or @jbevis Write a section of how to add media in it?

**Comment by perkinsjr** [comment]
@logan-anderson that would be great!

**Comment by logan-anderson** [comment]
@jbevis If you have time you can add it in. If not I can do it tomorrow morning.

File: content/guides/tina-cloud/starter/local.md [hunk 3]
```diff
@@ -31,25 +31,18 @@ cp .env.local.sample .env.local
...
-This is ideal for development workflows and the API is identical to the one used in the cloud, so once you're ready to deploy your application you won't face any challenges there.
+This is ideal for development workflows and the API is identical to the one used when working with the backend, so once you're ready to deploy your application you won't face any challenges there.
 
-Open [`http://localhost:3000`](http://localhost:3000) in your browser to see your file-based content being loaded from the GraphQL API.
+Open <a href="http://localhost:3000" target="_blank">http://localhost:3000</a> in your browser to see your file-based content being loaded from the GraphQL API.
 
 ## Edit content locally
 
-In the right-hand corner, click the "Edit this page" button. You'll see the page refresh and Tina will be enabled.
+Click on the pencil in the lower left-hand corner to open Tina's sidebar which displays a variety of forms with fields you can edit and see updates live on the page. Since we're working locally, clicking save results in changes to your local filesystem.
```
Comment (logan-anderson) [inline content/guides/tina-cloud/starter/local.md:46]: They will also have to enter edit mode by going to http://localhost:3000/admin

**Review by logan-anderson — CHANGES_REQUESTED** [review]

**Review by perkinsjr — APPROVED** [review]

**Review by perkinsjr — APPROVED** [review]
LGTM

**Review by logan-anderson — APPROVED** [review]
🎉

**Pull request merged**